### ISSUE TITLE
final peer review az529

### ISSUE THREAD

**Issue by Bible-Bullet** [jhmontijo/ORIE4741-Jackson-Alex-Kais#7]
This project focuses on predicting whether a flight would be delayed and the duration time. The dataset is quite of high quality, which is from the Bureau of Transportation Statistics and the National Oceanic and Atmospheric Administration. 

Things I like:
1. Very applicable topic. Actually it is a big problem for airlines to decrease the delay. You guys offered a reasonable model to deal with it, from which you could make a profit.

2. You mentioned the pros and cons of each algorithm and offered solutions for overfitting.

3. Great visualization and compose.

4. Daily to hourly data.

Possible improvement:
1. List the name and type of the features you apply and make a brief introduction.

2.  Maybe you could try more different loss functions and regularizers.

3. Think about fairness and WMD.
